compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/descriptors/WrappedDescriptors.kt

```diff
@@ -570,11 +672,11 @@ open class WrappedClassDescriptor(
 
     override fun getSource() = sourceElement
 
-    override fun getConstructors() = owner.declarations.asSequence().filterIsInstance<IrConstructor>().map { it.descriptor }.toList()
+    override fun getConstructors() =
+        owner.declarations.filterIsInstance<IrConstructor>().filter { !it.origin.isSynthetic }.map { it.descriptor }.toList()
```

Comment:
There are 10+ cases that failed with visibility conversion error: package-private (which should be normalized in general) doesn't have a counter part. By comparing `psi2ir` path, I noticed that all those cases have private constructor with default values, which is lowered by adding a synthetic constructor with package-private visibility.

The difference is, during serialization, `LazyClassDescriptor#getConstructors` returns constructors that literally appear in the PSI, which means we need to filter out synthetic ones here.